### ISSUE TITLE
1Password 8 disables Karabiner-Elements remappings

### ISSUE THREAD

**Issue by eddyg** [pqrs-org/Karabiner-Elements#3075]
The latest version of 1Password 8 has the ability to "auto-fill anything" (_including_ system security prompts).

Once I use 1Password 8 to fill a password, the keyboard remappings I have defined in Karabiner-Elements stop working.

If I quit 1Password 8 _completely_, my remappings start working again (until 1Password auto-fills something...)

**Comment by samplue** [comment]
Yes, same behavior here. I’ve also send feedback to 1password to see if it’s on their end

**Comment by kdevnel** [comment]
For anyone else coming this way, this forum post covers the issue - https://1password.community/discussion/129523/1password-autofill-breaks-hyperkey-config-in-karabiner-elements-bettertouchtool

tl;dr - it seems updating to macOS 12.4 solves the issue for some reason. It worked for me after the update though I don't know why.
